taxi/test/test_forms.py

```diff
@@ -0,0 +1,146 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.forms import (
+    CarModelSearchForm,
+    ManufacturerNameSearchForm,
+    DriverLicenseUpdateForm,
+    DriverUsernameSearchForm,
+    DriverCreationForm
+)
+from taxi.models import Manufacturer, Driver
+
+
+class TestCarForms(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="new_driver1",
+            password="root1234",
+        )
+        self.client.force_login(self.user)
+
+        self.manufacturer = Manufacturer.objects.create(
+            name="Manufacture Test", country="China"
+        )
+
+    def test_car_search_form_with_arg(self):
+        form_data = {
+            "model": "X3",
+        }
+        form = CarModelSearchForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+    def test_car_search_form_without_arg(self):
+        form_data = {
+            "model": "",
+        }
+        form = CarModelSearchForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+
+class TestManufacturerForms(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="new_driver1",
+            password="root1234",
+        )
+        self.client.force_login(self.user)
+
+        self.manufacturer1 = Manufacturer.objects.create(
+            name="Manufacture Test", country="China"
+        )
+        self.manufacturer2 = Manufacturer.objects.create(
+            name="Another Manufacturer", country="USA"
+        )
+        self.manufacturer3 = Manufacturer.objects.create(
+            name="Test Company", country="Germany"
+        )
+
+    def test_manufacturer_search_form_with_arg(self):
+        form_data = {
+            "name": "Test",
+        }
+        form = ManufacturerNameSearchForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+        manufacturers = Manufacturer.objects.filter(
```

Comment:
to correctly test search functionality you need to make a request to client with search queryand then check if results are correct:
```
response = self.client.get(url, {"search": "search_string"})
self.assertEquals(response.status_code, 200)
self.assertContains(response, "search_string")
self.assertNotContains(response, "another_string")